gm4_mountaineering/data/gm4_mountaineering/functions/pole_effects/poles_equipped.mcfunction

```diff
@@ -0,0 +1,16 @@
+# Give pole-related effects - jump boost on a cooldown and reduced fall damage
+# @s = player holding poles
+# at @s
+# run from player_main
+
+# Add jump boost effect (1 with skis, 2 without skis, 3 if having an upwards velocity)
+execute if score @s gm4_mountaineering_poles_jump_boost matches ..0 if score @s gm4_mountaineering_vy matches 2.. run effect give @s jump_boost 1 2 true
```

Comment:
```suggestion
effect give @s[scores={gm4_mountaineering_poles_jump_boost=..0, gm4_mountaineering_vy=2..}] jump_boost 1 2 true
```

Reply:
It's best to use as few execute subcommands as possible. I'll let you figure out how to fix the rest in this function